R/ggdensitree.R

```diff
@@ -59,7 +60,31 @@ ggdensitree <- function(data=NULL, mapping=NULL, layout="slanted", tip.order='md
 	
 	## determine tip order
 	if (length(tip.order) == 1) {
-		if (grepl('mds', tip.order)) {
+	  if (tip.order == 'mode') {
+	    first.label <- trees.f[[1]] %>%
+	      dplyr::filter(isTip) %>%
+	      dplyr::arrange(y) %>%
+	      dplyr::pull(label)
+
+		  res <- lapply(
+	      trees.f,
+	      . %>%
+	        dplyr::filter(isTip) %>%
+	        dplyr::arrange(y) %>%
+	        `$`("label") %>%
```

Comment:
.data[["label"]] should be equivalent and more readable IMO.

Reply:
I couldn't figure out how to add `.data[["label"]]` so I changed

```R
`$`('label") %>%
```

to

```R
dplyr::pull("label") %>%
```